src/SonarQube.Client.Tests/Models/ServerExclusionsTests.cs

```diff
@@ -112,5 +114,120 @@ public void Equals_SameReference_True()
 
             testSubject.Equals(testSubject).Should().BeTrue();
         }
+
+        [TestMethod]
+        public void Ctor_HasExclusions_AppendPathPrefix()
+        {
+            var testSubject = new ServerExclusions(
+                exclusions: new[] { "path1", "**\\path2", "**/path3", "**path4", "*/*" },
+                globalExclusions: Array.Empty<string>(),
+                inclusions: null);
+
+            testSubject.Exclusions.Should().BeEquivalentTo(
+                "**\\path1",
+                "**\\path2",
+                "**/path3",
+                "**\\**path4",
+                "**\\*/*");
+            testSubject.GlobalExclusions.Should().BeEmpty();
+            testSubject.Inclusions.Should().BeEmpty();
+        }
+
+        [TestMethod]
+        public void Ctor_HasGlobalExclusions_AppendPathPrefix()
+        {
+            var testSubject = new ServerExclusions(
+                exclusions: null,
+                globalExclusions: new[] { "path1", "**\\path2", "**/path3", "**path4", "*/*" },
+                inclusions: Array.Empty<string>());
+
+            testSubject.Exclusions.Should().BeEmpty();
+            testSubject.GlobalExclusions.Should().BeEquivalentTo(
+                "**\\path1",
+                "**\\path2",
+                "**/path3",
+                "**\\**path4",
+                "**\\*/*");
```

Comment:
I asked Caba about this, and they do it per char at a time, so it doesn't matter if some are forward and some are back.
https://github.com/SonarSource/sonar-dotnet/blob/a994db89dfb6136f75865ebc66306a19e5f23e76/analyzers/src/SonarAnalyzer.Common/Helpers/WildcardPatternMatcher.cs#L87

Reply:
Thinking about it again, it seems a bit weird that they support syntax that isn't supported by SonarQube/SonarCloud. I think we should stick to adding only documented syntax.